CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # 7.4.4
-- Bump promises dependency #46
+- Bump Promises dependency. (#46)
```

Comment:
Perhaps change #46 to https://github.com/firebase/firebase-ios-sdk/issues/8334.

Also, let's make this 7.5.0 since the major version dependency update could be destabilizing for clients who have separate Promises deps.